## 75% of end-end tests have passed
```
Total executed tests: 8
Failed tests: 1
Expected to fail tests: 1
Passed tests: 6
```
```
IDs of failed tests: 703133 
```
```
IDs of expected to fail tests: 702843 
```
<h3>Failed tests (1)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=chore/remove-notifications-screen&PR_ID=21633&APK_NAME=21633.apk&TR_CASE_IDS=703133">Rerun failed tests</a></li><br/><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3725956">test_restore_multiaccount_with_waku_backup_remove_profile_switch</a>, id: 703133 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Button` by `accessibility id`: `show-profiles` </div><div>Device 1: Tap on found: Button </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:245: in test_restore_multiaccount_with_waku_backup_remove_profile_switch
    self.errors.verify_no_errors()
base_test_case.py:192: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 zQ3...dWXh5 was not restored as a contact from waku backup!
E    zQ3...Vacac was not restored as a contact from waku backup!
E    admin_open was not restored from waku-backup!!
E    member_open was not restored from waku-backup!!
E    admin_closed was not restored from waku-backup!!
E    member_closed was not restored from waku-backup!!
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fd722cca850e4186a9081530dfa16799?auth=8f2b470347b1a24108ab093cda2bb4c4#79">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Expected to fail tests (1)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3725955">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td>

```
Test is not run, e2e blocker  
```

<b>[[reason: [NOTRUN] Skipped due to waku issue on staging fleet]]</b><br/><br/></td></tr></tbody></table></details><h3>Passed tests (6)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3725953">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fd722cca850e4186a9081530dfa16799?auth=8f2b470347b1a24108ab093cda2bb4c4#47">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3725959">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3725960">test_wallet_balance_mainnet</a>, id: 740490 </b></td></tr><tr><td></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3725954">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/371f2405e7514897855a11198272222b?auth=75464adc5542d1b70eca7c83c1e1ce77#100">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1984d22967a0499599cae25d2d0c4811?auth=9cbbabd399532dfec45e3d5d2199fde0#95">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3725958">test_wallet_send_asset_from_drawer</a>, id: 727230 </b></td></tr><tr><td></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3725957">test_wallet_send_eth</a>, id: 727229 </b></td></tr><tr><td></td></tr></tbody></table></details>

--- a/src/status_im/contexts/profile/login/events.cljs
+++ b/src/status_im/contexts/profile/login/events.cljs
@@ -84,10 +84,7 @@
 
                   (get db :onboarding/new-account?)
                   [[:dispatch [:onboarding/finalize-setup]]
-                   [:dispatch
-                    [:navigate-to-within-stack
-                     [:screen/onboarding.enable-notifications
-                      :screen/onboarding.preparing-status]]]]
+                   [:dispatch [:onboarding/finish-onboarding false]]]
 
                   :else
                   [[:dispatch [:update-theme-and-init-root :shell-stack]]

--- a/test/appium/views/profile_view.py
+++ b/test/appium/views/profile_view.py
@@ -237,7 +237,8 @@ class ProfileView(BaseView):
         self.delete_profile_button = Button(self.driver, accessibility_id="delete-profile-confirm")
 
         # Notifications
-        self.profile_notifications_button = Button(self.driver, accessibility_id="notifications-settings-button")
+        self.profile_notifications_button = Button(self.driver,
+                                                   accessibility_id="icon, Notifications, label-component, icon")
         self.profile_notifications_toggle_button = Button(self.driver,
                                                           accessibility_id="local-notifications-settings-button")
         self.push_notification_toggle = Button(
@@ -389,7 +390,7 @@ class ProfileView(BaseView):
         self.driver.info("Enabling push notifications via Profile")
         self.profile_notifications_button.scroll_and_click()
         self.profile_notifications_toggle_button.click()
-        self.navigate_back_to_home_view()
+        self.allow_button.click_if_shown()
 
     def add_custom_network(self, rpc_url: str, name: str, symbol: str, netwrok_id: str):
         self.driver.info("## Add custom network", device=False)

--- a/test/appium/views/sign_in_view.py
+++ b/test/appium/views/sign_in_view.py
@@ -214,8 +214,6 @@ class SignInView(BaseView):
         self.profile_confirm_password_button = Button(self.driver, translation_id="password-creation-confirm")
         self.enable_biometric_maybe_later_button = Button(self.driver, translation_id="maybe-later")
         self.identifiers_button = Button(self.driver, accessibility_id="skip-identifiers")
-        self.enable_notifications_button = Button(self.driver, accessibility_id="enable-notifications-button")
-        self.maybe_later_button = Button(self.driver, accessibility_id="enable-notifications-later-button")
         self.start_button = Button(self.driver, accessibility_id="welcome-button")
         self.use_recovery_phrase_button = Button(self.driver, accessibility_id="use-recovery-phrase-option-card")
         self.passphrase_edit_box = EditBox(self.driver, accessibility_id="passphrase-input")
@@ -240,10 +238,8 @@ class SignInView(BaseView):
         if set_image:
             pass
 
-    def create_user(self, password=common_password, keycard=False, enable_notifications=False,
-                    username="test user", first_user=True):
-        self.driver.info("## Creating new multiaccount (password:'%s', keycard:'%s', enable_notification: '%s')" %
-                         (password, str(keycard), str(enable_notifications)), device=False)
+    def create_user(self, password=common_password, username="test user", first_user=True, enable_notifications=False):
+        self.driver.info("## Creating new multiaccount with password:'%s'" % password, device=False)
         if first_user:
             self.create_profile_button.click_until_presence_of_element(self.generate_keys_button)
             self.not_now_button.wait_and_click()
@@ -255,40 +251,18 @@ class SignInView(BaseView):
         self.generate_keys_button.click_until_presence_of_element(self.profile_title_input)
         self.set_profile(username)
         self.set_password(password)
-        # if self.enable_biometric_maybe_later_button.is_element_displayed(10):
-        #     self.enable_biometric_maybe_later_button.click()
-        # self.next_button.click_until_absense_of_element(self.element_by_translation_id("intro-wizard-title2"))
-        # if keycard:
-        #     keycard_flow = self.keycard_storage_button.click()
-        #     keycard_flow.confirm_pin_and_proceed()
-        #     keycard_flow.backup_seed_phrase()
-        # else:
-        #     self.next_button.click()
-        #     self.create_password_input.send_keys(password)
-        #     self.confirm_your_password_input.send_keys(password)
-        #     self.next_button.click()
-        # self.identifiers_button.wait_and_click(30)
-        if enable_notifications:
-            self.enable_notifications_button.wait_and_click()
-            for _ in range(3):
-                self.allow_button.click_if_shown(sec=10)
-                self.enable_notifications_button.click_if_shown()
-                if self.chats_tab.is_element_displayed():
-                    break
-        else:
-            self.maybe_later_button.wait_and_click()
-            for _ in range(3):
-                self.maybe_later_button.click_if_shown()
-                if self.chats_tab.is_element_displayed():
-                    break
         self.chats_tab.wait_for_visibility_of_element(30)
         self.driver.info("## New multiaccount is created successfully!", device=False)
-        return self.get_home_view()
+        home_view = self.get_home_view()
+        if enable_notifications:
+            profile_view = home_view.profile_button.click()
+            profile_view.switch_push_notifications()
+            profile_view.click_system_back_button(times=2)
+        return home_view
 
-    def recover_access(self, passphrase: str, password: str = common_password, keycard=False,
-                       enable_notifications=False, second_user=False, username='Restore user', set_image=False,
-                       after_sync_code=False):
-        self.driver.info("## Recover access(password:%s, keycard:%s)" % (password, str(keycard)), device=False)
+    def recover_access(self, passphrase: str, password: str = common_password, enable_notifications=False,
+                       second_user=False, username='Restore user', set_image=False, after_sync_code=False):
+        self.driver.info("## Recover access (password:%s)" % password, device=False)
 
         if not after_sync_code:
             if not second_user:
@@ -304,22 +278,13 @@ class SignInView(BaseView):
         if not after_sync_code:
             self.set_profile(username, set_image)
         self.set_password(password)
-        if enable_notifications:
-            self.enable_notifications_button.wait_and_click()
-            for _ in range(3):
-                self.allow_button.click_if_shown(sec=10)
-                self.enable_notifications_button.click_if_shown()
-                if self.chats_tab.is_element_displayed():
-                    break
-        else:
-            self.maybe_later_button.wait_and_click()
-            for _ in range(3):
-                self.maybe_later_button.click_if_shown()
-                if self.chats_tab.is_element_displayed():
-                    break
         self.chats_tab.wait_for_visibility_of_element(30)
         self.driver.info("## Multiaccount is recovered successfully!", device=False)
-        return self.get_home_view()
+        home_view = self.get_home_view()
+        if enable_notifications:
+            profile_view = home_view.profile_button.click()
+            profile_view.switch_push_notifications()
+        return home_view
 
     def sync_profile(self, sync_code: str, first_user: bool = True):
         if first_user:

